client/state/sharing/keyring/selectors.js

```diff
@@ -17,7 +17,7 @@ import 'calypso/state/sharing/init';
  * @returns {Array}        Keyring connections, if known.
  */
 export function getKeyringConnections( state ) {
-	return values( state.sharing.keyring.items );
+	return Object.values( state.sharing.keyring.items );
```

Comment:
Not sure how to audit this either.

Reply:
It's always an object, it defaults to an empty object at the reducer default state: https://github.com/Automattic/wp-calypso/blob/trunk/client/state/sharing/keyring/reducer.js#L36